logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
> We only want to create one extra event with the debugging information and that should be the first one in the series of events

I disagree - when you are debug mode I think the debug should be per event.

I think this emphasises that instead of hardcoding the debug info we should probably provide a hook and drupal_alter()  when in debug mode.

So for example I would add the debug per event (i.e. use the ``drupal_alter()``) while you will probably use the hook to register a new event using ``logs_http_register_event()``). 

What do you say? :)